src/appleseed-max-impl/appleseedoslplugin/oslshadermetadata.h

```diff
@@ -162,7 +158,7 @@ class OSLParamInfo
     double softMaxValue;
     bool divider;
 
-    bool mayaAttributeConnectable;
+    bool connectable;
```

Comment:
`m_connectable`

In fact, all class members, methods and free functions should begin with `m_` and should be spelled in snake case, e.g. `m_soft_max_value`.